tests/python/test_simulation_analysis.py

```diff
@@ -12,126 +19,75 @@ def app():
     return app
 
 
+test_impact = {
```

Comment:
Move this and test_json to tests/fixtures/simulation_analysis_fixtures.py to reduce duplication in test files and make it easier to maintain consistent test data across multiple test cases.